src/lib/command_line.ml

```diff
@@ -58,6 +58,14 @@ let client ~base_url action ids =
     List.iter statuses ~f:(fun (r, s) ->
       printf "%s is %s\n" r (Kube_job.Status.show s))
     |> return
+  | `List ->
+    Client.get_job_list client
+    >>= fun jobs ->
+    List.iter jobs ~f:(fun (`Id i, `Status s)  ->
+      printf "%s is %s\n" i s)
+    |> return
```

Comment:
it would be clearer:

```ocaml
   printf "%s is %s\n" i s);
return ()
```

the `|>` makes it look like you want to return something